apps/broker-service/src/controllers/Newsletter.controller.ts

```diff
@@ -1,94 +1,75 @@
-import { log } from '@thecointech/logging';
 import { Controller, Get, Put, Route, Query, Body, Post, Response, Tags } from '@tsoa/runtime';
-import { Signup, Unsubscribe, Details, SubscriptionDetails, Update } from '../newsletter'
+import { signup, unsubscribe, details, update } from '../newsletter'
+import { SubscriptionData, Subscription } from '@thecointech/broker-db';
 import { BoolResponse } from '../types';
+import { ErrorResponse, SubscriptionDetails } from './types';
+
+// Convert from datetime rep to number timestamp
+function convert(sub?: Subscription) {
+  if (!sub) return { error: "Not Found" };
+  const { registerDate, ...rest } = sub;
+  return {
+    registerDate: registerDate.toMillis(),
+    ...rest,
+  }
+}
```

Comment:
Eventually, all our code should use exclusively `DateTime`.  Not `Timestamp`, not `Date`, and not `number`.  Any conversions happen at the very leaves, and nobody else should be aware that they are happening.